### PR TITLE
Allow config to set the interval a cdn is resynced

### PR DESCRIPTION
**Pull request by megawac**

### CODE REVIEW

File: config/config.template.js [hunk 1]
```diff
@@ -19,10 +19,11 @@ module.exports = {
...
   },
   tasks: {
-    "bootstrap-cdn": {minute: 3},
-    "google": {minute: 3},
-    "cdnjs": {minute: 3},
-    "jsdelivr": {minute: 3}
-    //jquery: {minute: 5}
+    // resync interval is in seconds
+    "bootstrap-cdn": {interval: 180},
+    "google": {interval: 180},
```
Comment (jimaek) [inline config/config.template.js]: Google updates rarely and we dont want to get banned. 30 minutes?

File: config/config.template.js [hunk 2]
```diff
@@ -19,10 +19,11 @@ module.exports = {
...
+    "cdnjs": {interval: 180},
+    "jsdelivr": {interval: 180}
```
Comment (jimaek) [inline config/config.template.js]: 15 minutes sounds good for both cdnjs and jsdelivr?

File: config/config.template.js [hunk 3]
```diff
@@ -19,10 +19,11 @@ module.exports = {
     email: ''
   },
   tasks: {
-    "bootstrap-cdn": {minute: 3},
-    "google": {minute: 3},
-    "cdnjs": {minute: 3},
-    "jsdelivr": {minute: 3}
-    //jquery: {minute: 5}
+    // resync interval is in seconds
+    "bootstrap-cdn": {interval: 180},
```
Comment (jimaek) [inline config/config.template.js]: I guess we need to move it to local scan as well?

Comment (megawac) [inline config/config.template.js]: Why's that? there's not too many projects on bootstrap-cdn, I didn't think it was worth the effort

Comment (jimaek) [inline config/config.template.js]: ok then

**Comment by jimaek** [comment]
Is this still good to go? Is it compatible with the local api-sync?

**Comment by megawac** [comment]
Should be

**Comment by jimaek** [comment]
Please check. Fix the conflicts, test and merge if it works

**Comment by megawac** [comment]
Rebased this

**Pull request merged**